site/content/en/docs/integration/_index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Integration"
```

Comment:
I don't like the topic. From my perspective integrations are more about collaboration with external tools (see https://voxel51.com/docs/fiftyone/integrations/index.html). The topic about our REST API, SDK, and CLI. Probably the best option is to just have 3 separate topics: REST API, SDK, CLI. Look at https://docs.labelbox.com/docs as well. I like the documentation structure.

Reply:
I can see all the variants in different tools:
- Labelbox: API & SDK
- 51: API, CLI
- Label studio: Integrations (API, SDK, CLI)
- Superannotate: SDK (+CLI); Integrations (with others + their dataset format)
- v7: SDK; CLI; API

Probably, I agree about separate sections for SDK and CLI, but not sure about API.

Reply:
Changed section name to `API & SDK`. The `API & SDK & CLI`, `API, SDK & CLI` variants look clumsy to my taste, and SDK can also suppose CLI by its meaning, so it shouldn't be a big loss.